ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -29,13 +30,15 @@ component:
   dial: "Adjustment device (e.g. setpoint dial)."
   dimmer: "Adjustment device of dimming light level for controlling output of a lighting fixture."
   dishwasher: "Device that washes dishes."
+  driver: "A circuit or component used to control another circuit or component, such as a high-power transistor, light-emitting diode(LED), and numerous others.
```

Comment:
Is this different from controller above? If not, we should choose one to use and remove the other.

Reply:
Yes it is different from controller. @pisuke recommended to add "driver" for LED to differentiate with "ballast" which is only used for fluorescent light.

Reply:
@tasodorff @pisuke any comments?

Reply:
this sounds like `controller` to me, just specific to the lighting domain. `driver` has multiple meanings outside this (e.g. a network driver). 

if you were to distinguish it from a controller, how would you?

Reply:
@JennyChan-ptpl friendly ping :)

Reply:
@charbull A motor controllers provide feedback, error detection, and other features which usually aren't found on basic motor driver boards.

Reply:
So then we should adjust the driver definition to explicitly state that it provides the motor (or lighting board) with power, but doesn't handle logical control of the device.

Reply:
Thanks for your suggestion, the driver definition had been updated.
driver: "A circuit or component that provides the motor (or lighting board) with power, but doesn't handle logical control of the device."

Reply:
Any further updates needed?

Reply:
@tasodorff , If you have no further comments, can you help to merge to the master copy?